src/kvstore/kvstore_dist.h

```diff
@@ -636,7 +648,7 @@ class KVStoreDist : public KVStoreLocal {
   // Note: this encoding method for row sparse keys doesn't allow cross-layer batching
   inline PSKV& EncodeRowSparseKey(const int key, const int64_t size, const int64_t num_rows,
                                   const int64_t *offsets, const size_t unit_len,
-                                  const int64_t total_num_rows) {
+                                  const int64_t total_num_rows, int num_bytes) {
```

Comment:
nit: `const int num_bytes` (everything else is -- make it look like this has a special meaning since it isn't)

Reply:
throughout, yes pls try to reduce the superfluous computations